src/datachain/studio.py

```diff
@@ -139,3 +152,69 @@ def save_config(hostname, token):
         conf["studio"] = studio_conf
 
     return config.config_file()
+
+
+def create_job(
+    query_file: str,
+    team_name: Optional[str],
+    env_file: Optional[str] = None,
+    envs: Optional[list[str]] = None,
+    workers: Optional[int] = None,
+    files: Optional[list[str]] = None,
+    python_version: Optional[str] = None,
+    reqs: Optional[str] = None,
+    req_file: Optional[str] = None,
+):
+    query_type = "PYTHON" if query_file.endswith(".py") else "SHELL"
```

Comment:
This might be a problem in future. I think we should let user choose file type either by additional argument or even by separate command.